starter-workflows/README.md

```diff
@@ -0,0 +1,5 @@
+**Code Scanning Workflows**
+
+GitHub code scanning is a developer-first, GitHub-native approach to easily find security vulnerabilities before they reach production.
+Whenever you make changes to the files in this directory, you should update them in this repository as well:
```

Comment:
Let's rephrase something explaining that this sentence is for Frogbot developers only, for example:
```suggestion
## Frogbot Contributors
Whenever you make changes to the files in this directory, you should update them in this repository as well:
```

Reply:
done